.gitignore

```diff
@@ -3,4 +3,6 @@ __pycache__
 .emacs.desktop*
 tests/data/.*
 src/.data
-.vscode
\ No newline at end of file
+.vscode
+*.gz
```

Comment:
Should these be in `.gitignore`?

Reply:
I think some *.gz files under tests/data maybe used for tests

Reply:
I believe those are already ignored by the line `tests/data/.*`